src/zenml/stack/stack.py

```diff
@@ -386,7 +386,7 @@ def resume(self) -> None:
             ProvisioningError: If any stack component is missing provisioned
                 resources.
         """
-        logger.info("Resuming provisioned resources for stack '%s'.", self.name)
+        logger.info(f"Resuming provisioned resources for stack '{self.name}'.")
```

Comment:
I think where possible, we use %s in logger.

Reply:
Reverted.